.github/workflows/quarto-publish.yml

```diff
@@ -19,6 +19,10 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v4
 
+      - name: Overwrite _quarto.yml
+        run: |
+          cp .github/_quarto.yml _quarto.yml
```

Comment:
Isn't this the wrong way round, copying the yml file with references to `maehr/open-research-data-template` to the file in my own project?

Reply:
You are right, there is an if clause missing that makes sure we are in the template repo. Thanks